pkg/udev/common_test.go

```diff
@@ -199,3 +206,19 @@ func TestGetDevLinks(t *testing.T) {
 		}
 	}
 }
+
+func unorderedEqual(first, second []string) bool {
```

Comment:
Could you add a comment and some examples of what is expected out of this function?

Reply:
Done.